packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -59,4 +60,155 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * The IconProps to use for the button aspect of the combobox
    */
   buttonIconProps?: IIconProps;
+
+  /**
+   * Theme provided by HOC.
+   */
+  theme?: ITheme;
+
+  /**
+   * Custom styles for this component
+   */
+  styles?: IComboBoxStyles;

```

Comment:
Let's change the way we do this.
For this one, have it be Partial<IComboBoxStyles>
Then in IComboBoxStyles, make everything required instead of ?-optional.
(I'll do this for the rest of our glamorized controls when I have some time.)
This way, the compiler will warn us if we forget to provide a default style for something in IComboBoxStyles in ComboBox.styles.ts, while users can still pass in a partial implementation of the interface for customization.